cmd/system.go

```diff
@@ -15,12 +15,20 @@ var (
 	deployConfig    cfg.DeployConfig
 )
 
+const (
+	DCETFInitOptionsEnvVar  string = "DCE_TF_INIT_OPTIONS"
+	DCETFApplyOptionsEnvVar string = "DCE_TF_APPLY_OPTIONS"
+	Empty                   string = ""
+)
+
 func init() {
 	deployOverrides = svc.DeployOverrides{}
 	deployConfig = cfg.DeployConfig{}
 	systemDeployCmd.Flags().StringVarP(&deployConfig.DeployLocalPath, "local", "l", "", "Path to a local DCE repo to deploy.")
 	systemDeployCmd.Flags().BoolVarP(&deployConfig.UseCached, "use-cached", "c", true, "Overwrite local backend state.")
 	systemDeployCmd.Flags().BoolVarP(&deployConfig.BatchMode, "batch-mode", "b", false, "Skip prompting for resource creation.")
+	systemDeployCmd.Flags().StringVar(&deployConfig.TFInitOptions, "tf-init-options", "", "Options to pass to the underlying `tf init` command.")
```

Comment:
What's the format of these options? I had seen JSON at one point... 
Or are the options just passed through as-is to the TF commands (formatted like `--tf-apply-option "-input=false -var foo=bar"`)

Reply:
Good point above re: documentation which would also clear this question up. You're guessing correctly here.